docs/operating-scylla/security/certificate-authentication.rst

```diff
@@ -0,0 +1,52 @@
+Certificate Based Authentication
+================================
+
+Certificate based authentication is an alternative method of resolving a login to credentials, i.e. a defined ScyllaDB role.
+Instead of defining one or more user/password pairs, each defined role is assigned depending on verified transport 
+client certificate Subject/Alt information.
+
+
+Procedure
+---------
+
+#. Enable authentication and define authorized roles in the cluster as described in the 
+`Enable Authentication </operating-scylla/security/authentication/>` document. 
+
+#. Enable CQL transport TLS using client certificate verification in each node.
+
+.. code-block:: yaml
+
+   client_encryption_options:
+      enabled: True
+      certificate: <server cert>
+      keyfile: <server key>
+      truststore: <shared trust>
+      require_client_auth: True
+
+Create the client certificates such that each one designates the desired login role for this connection. A typical method 
+would be to assign the subject common name as the role name. All client certificates need to be signed in such a way that 
+either system trust or the defined trust store can verify them.
+
+#. Configure the certificate based authenticator in each node.
+
+.. code-block:: yaml
+
+   authenticator: com.scylladb.auth.CertificateAuthenticator
+   auth_certificate_role_queries:
+            - source: SUBJECT
+              query: CN=([^,\s]+)
+
+More than one expression can be used to extract role names. The first one mathing wil be used. The available sources are `SUBJECT` (distinguished name) and
+`ALTNAME` (subject extensions). The query should be a regular expression with a single match clause.
+
+Restart the cluster.
```

Comment:
```suggestion
   .. code-block:: yaml

      authenticator: com.scylladb.auth.CertificateAuthenticator
      auth_certificate_role_queries:
               - source: SUBJECT
                 query: CN=([^,\s]+)

   More than one expression can be used to extract role names. The first one mathing wil be used. The available sources are `SUBJECT` (distinguished name) and
   `ALTNAME` (subject extensions). The query should be a regular expression with a single match clause.

Restart the cluster.
```

Reply:
These changes are required to fix the numbered list

Reply:
Fixed